views/form.blade.php

```diff
@@ -14,9 +14,12 @@
             @if(!$tabObj->isEmpty())
                 @include('admin::form.tab', compact('tabObj'))
```

Comment:
I disagree, it is suppose to be simple, horizontal is a basic thing. tabs for fields are over complicated.
BTW, sleeping owl has the worst API for horizontal fields